.github/workflows/rust_test.yml

```diff
@@ -5,17 +5,6 @@ on:
   workflow_call:
 
 jobs:
-  fmt:
-    name: fmt
-    runs-on: ubuntu-latest
-    timeout-minutes: 30
-    steps:
-      - uses: actions/checkout@v4
-      - uses: dtolnay/rust-toolchain@nightly
-        with:
-          components: rustfmt
-      - run: cargo fmt --all --check
-
```

Comment:
Are we sure that the trunk fmt check is done properly with cargo nightly? because this is important to sort the imports properly in each file, which is not done in non-nightly mode

Reply:
copied from last keth update, trunk uses the system installed (meaning the project) rust

Reply:
confirming that this
<img width="269" alt="image" src="https://github.com/user-attachments/assets/308bf6ec-d61d-45b3-95de-e1c6db31ebdd">
is formatted into 
<img width="211" alt="image" src="https://github.com/user-attachments/assets/a3f609f5-e84c-4037-b8be-64a7e090f603">
by `trunk fmt`

Reply:
nice